sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -174,6 +174,16 @@ export type GetTableEntityResponse<T extends object> = TableEntity<T> & {
     };
 };
 
+// @public
+export const enum KnownGeoReplicationStatusType {
+    // (undocumented)
```

Comment:
This line of code change is not really related to Version Upgrade. I just found a forgotten export and added it. Please let me know if this is not exported intentionally. I can revert the changes. Thanks